Explain dynamic ad insertion (#224)
AntennaPod/antennapod.github.io: _i18n/en/documentation/general/ads.md

> The official versions of AntennaPod **do not insert any ads**. You can even verify this because [AntennaPod's source code is public](https://github.com/AntennaPod/AntennaPod). Apart from that, AntennaPod is developed by volunteers without commercial interest.

Reviewer: ```suggestion
The official versions of AntennaPod do not insert any ads. The app is developed by volunteers without commercial interest and the project relies on donations to cover costs. As AntennaPod's [source code](https://github.com/AntennaPod/AntennaPod) and [income and expenses](https://opencollective.com/antennapod#category-BUDGET) are public everyone is free to verify this.
```
I can't verify "can" not verify this in code code because I don't have the skills - hence 'free to'.

If you are seeing ads, this can have two reasons:

### 1. You downloaded an unofficial version of AntennaPod

> AntennaPod is open-source. This means that anyone can download the source code. Unfortunately, some developers take AntennaPod, insert ads and then publish it as a new app. You can get the official AntennaPod app on [Google Play](https://play.google.com/store/apps/details?id=de.danoeh.antennapod) or [F-Droid](https://f-droid.org/packages/de.danoeh.antennapod/). You can even get the latest release on [GitHub](https://github.com/AntennaPod/AntennaPod/) and build AntennaPod yourself!

Reviewer: ```suggestion
AntennaPod is open-source. This means that anyone can download the source code. Unfortunately, some developers take AntennaPod, insert ads and then publish it as a new app. You can get the official AntennaPod app on [Google Play](https://play.google.com/store/apps/details?id=de.danoeh.antennapod) and [F-Droid](https://f-droid.org/packages/de.danoeh.antennapod/). You can even get the latest release from [GitHub](https://github.com/AntennaPod/AntennaPod/) and build AntennaPod yourself!
```
Corrections while we're at it.

### 2. The podcast publishers added the ads to their shownotes or audio file

AntennaPod plays podcasts just like they were published by the creators. If they include ads, AntennaPod plays them. We have nothing to do with those ads, so the revenue fully goes to the publishers of your podcast.

> Now, you might ask something like "But then why is the podcast in English and the ad in German?" or "But why is the podcast episode from 2008 but still promotes a product released in 2023?". The reason is a technique called **dynamic ad insertion**. Podcast publishers can instruct the hosting company to insert those ads. Then, while downloading/streaming the episode with AntennaPod, the media file delivered to AntennaPod gets cut on-the-fly to include the ads. Depending on how smart the hosting company does this, the ads might get inserted right in the middle of a sentence. Some of the hosting companies also mess up the chapter marks while inserting the ads (so the chapters are off by the duration of the ad).

Reviewer: ```suggestion
Now, you might ask something like "But then why is the podcast in English and the ad in German?" or "But why is the podcast episode from 2008 but still promotes a product released in 2023?". The reason is a technique called **dynamic ad insertion**. Podcast publishers can instruct the hosting company to insert those ads. Then, while downloading or streaming the episode with AntennaPod, the media file delivered to AntennaPod gets cut on-the-fly to include the ads. The ads are recent (while the episode might not be) and can be targeted based on the IP address from which you request the media file (e.g. a company in your country). Depending on how smart the hosting company does this, the ads might get inserted right in the middle of a sentence. Some of the hosting companies also mess up the chapter marks while inserting the ads (so the chapters are off by the duration of the ad).
```

Trying to answer the questions you pose.
